lib/reek/configuration/rake_task_converter.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Reek
+  module Configuration
+    # Responsible for converting configuration values coming from the outside world
+    # to whatever we want to use internally.
+    module RakeTaskConverter
```

Comment:
This module seems tested but not used ...

Reply:
It is used at the rake task at the [very end](https://github.com/troessner/reek/pull/1378/files#diff-09c36c2af56d3a3b653a7c7a4e31bb71R12) of this PR ;)

Reply:
Ah, I don't know how I missed that.